contract/vm_callback.go

```diff
@@ -1492,6 +1500,10 @@ func luaCheckView(service C.int) C.int {
 	return C.int(ctx.nestedView)
 }
 
+// luaCheckTimeout checks two types of timeouts;
+// One is whether the block creation timeout is exceeded, and the other is whether
+// the transaction execution timeout is exceeded.
```

Comment:
; -> :
is -> was (2 times)

Reply:
Or like this:
```
// luaCheckTimeout checks two types of timeouts:
// 1. whether the block creation timeout was exceeded
// 2. whether the transaction execution timeout was exceeded
```

Reply:
Or use "occurred" instead:
```
// luaCheckTimeout checks two types of timeouts:
// 1. whether the block creation timeout occurred
// 2. whether the transaction execution timeout occurred
```

Reply:
You did not add the notes above. Is the function not checking 2 timeouts anymore?